Sources/Parsers/AssetsCatalogParser.swift

```diff
@@ -59,16 +60,20 @@ private enum AssetCatalog {
     static let providesNamespace = "provides-namespace"
   }
 
-  enum Item {
-    static let imageSet = "imageset"
+  enum Item: String {
+    case colorSet = "colorset"
+    case imageSet = "imageset"
 
     /**
      * This is a list of supported asset catalog item types, for now we just
-     * support `image set`s. If you want to add support for new types, just add
-     * it to this whitelist, and add the necessary code to the
-     * `process(items:withPrefix:)` method.
+     * support `image set`s and `color set`s. If you want to add support for
+     * new types, just add it to this whitelist, and add the necessary code to
+     * the `process(items:withPrefix:)` method.
+     *
+     * Use as reference:
+     * https://developer.apple.com/library/content/documentation/Xcode/Reference/xcode_ref-Asset_Catalog_Format
      */
-    static let supported = [imageSet]
+	static let supported: [Item] = [.colorSet, .imageSet]
```

Comment:
Now that we use `enum cases` then that `supported` constant is actually no longer needed!